xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -70,26 +70,6 @@ benchmarks! {
 
 	}
 
-	// Worst case scenario for this benchmark is a large number of assets to
-	// filter through the reserve.
-	reserve_asset_deposited {
-		const MAX_ASSETS: u32 = 100; // TODO when executor has a built in limit, use it here. #4426
-		let mut executor = new_executor::<T>(Default::default());
-		let assets = (0..MAX_ASSETS).map(|i| MultiAsset {
-			id: Abstract(i.encode()),
-			fun: Fungible(i as u128),
-
-		}).collect::<vec::Vec<_>>();
-		let multiassets: MultiAssets = assets.into();
-
-		let instruction = Instruction::ReserveAssetDeposited(multiassets.clone());
-		let xcm = Xcm(vec![instruction]);
-	}: {
-		executor.execute(xcm).map_err(|_| BenchmarkError::Skip)?;
-	} verify {
-		assert_eq!(executor.holding, multiassets.into());
-	}
-
```

Comment:
This whole benchmark needs to be removed?

There is already a `skip` benchmark error, trying to understand what the intention is

Reply:
This was spotted by Keith afair as a 100% duplicate of the same bench in fungible.